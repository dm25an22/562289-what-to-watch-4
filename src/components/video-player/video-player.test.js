import React from "react";
import rerender from "react-test-renderer";
import VideoPlayer from "./video-player.jsx";

const settings = {
  className: `player__video`,
  loop: true,
  muted: true,
  width: `280px`,
  height: `175px`
};

it(`Render VideoPlayer`, () => {
  const tree = rerender.create(
      <VideoPlayer
        preview={`https://upload.wikimedia.org/wikipedia/commons/transcoded/b/b3/Big_Buck_Bunny_Trailer_400p.ogv/Big_Buck_Bunny_Trailer_400p.ogv.360p.webm`}
        poster={`img`}
        settings={settings}
        isPlaying={true}
      />, {
        createNodeMock: () => {
          return {};
        }
      }).toJSON();

  expect(tree).toMatchSnapshot();
});
