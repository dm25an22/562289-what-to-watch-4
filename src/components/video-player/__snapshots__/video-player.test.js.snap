// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render VideoPlayer 1`] = `
<video
  className="player__video"
  height="175"
  loop={true}
  muted={true}
  width="280px"
/>
`;
